src/main/cpp/chassis/states/DriveToFieldElement.cpp

```diff
@@ -235,13 +226,13 @@ void DriveToFieldElement::CalculateFeedForward(ChassisMovement &chassisMovement)
 {
     if (m_chassis != nullptr)
     {
-        units::meter_t distance = m_currentPose.Translation().Distance(m_endPose.Translation());
+        m_distanceError = m_currentPose.Translation().Distance(m_endPose.Translation());
 
         // Calculate feedforward speed based on distance
         units::velocity::meters_per_second_t feedforwardSpeed = 0.0_mps;
-        if (distance > m_ffMinRadius)
+        if (m_distanceError > m_ffMinRadius)
         {
-            double feedForwardScale = std::clamp(((distance - m_ffMinRadius) / (m_ffMaxRadius - m_ffMinRadius)).value(), 0.0, 1.0);
+            double feedForwardScale = std::clamp(((m_distanceError - m_ffMinRadius) / (m_ffMaxRadius - m_ffMinRadius)).value(), 0.0, 1.0);
```

Comment:
Min and max are constants so make another costant that is diff so we don't need to calculate that part every loop

Reply:
Done